bindings/cpp/newapi/session.cpp

```diff
@@ -1107,20 +1107,20 @@ class server_send_handler : public std::enable_shared_from_this<server_send_hand
 			const auto &remote = pair.first;
 			const auto &ids = pair.second;
 
-			auto request = serialize(dnet_server_send_request{
-				ids,
-				dst_groups,
-				flags,
-				chunk_size,
-				chunk_write_timeout,
-				chunk_commit_timeout,
-				chunk_retry_count,
-			});
+			dnet_server_send_request request;
```

Comment:
Yes, in-place is better, but only if we discuss speed.

My reasons:
* Now all requests are inherited from some common request, each request has extra fields at beginning (how many we can't know: while develop progress count of fields can vary)
* `initializer_list`-like initialization isn't good way for big structs. Seeing the code, you cannot determine meaning of initializer without navigating to header and counting field position. Here constructor can be used, but the same problem with unnamed parameters passing.

Here it's not worth the trouble, because it isn't hotspot and performance gain will be inappreciable. Code readability benefit is evident.

Reply:
You can use comments as was in code before or initialization with dots:
```
my_type obj { .a = 1, .b = 2 };
```

Reply:
The main reason why in-place initialization is better:

When you add a new attribute to structure, the compilation will fail, while not all using of structure will be fixed .
This is very important for structures without default values for attributes and can save you from undefined behavior.

Reply:
At least call `memset(..., 0, ...);`